src/assets/i18n/cs.json5

```diff
@@ -4013,8 +4015,8 @@
   "search.filters.applied.f.discoverable": "Soukromé",
   // "search.filters.applied.f.entityType": "Item Type",
   "search.filters.applied.f.entityType": "Typ položky",
-  // "search.filters.applied.f.has_content_in_original_bundle": "Has files",
-  "search.filters.applied.f.has_content_in_original_bundle": "Má soubory",
+  // "search.filters.applied.f.has_content_in_original_bundle": "Contain Files",
```

Comment:
This would be correct: `Contains Files`

Reply:
Are you sure? I copied it from UFAL clarin DSpace: 
![image](https://github.com/dataquest-dev/dspace-angular/assets/90026355/70d5ad95-c0a9-42d9-b92d-df9b6f65eae3)